Source/iop/IopBios.cpp

```diff
@@ -120,6 +123,8 @@ CIopBios::~CIopBios()
 
 void CIopBios::Reset(const Iop::SifManPtr& sifMan)
 {
+	auto cdvdHandler = reinterpret_cast<uint32*>(&m_ram[BIOS_PSEUDO_INTRHANDLER_TABLE_BASE + (BIOS_PSEUDO_INTRHANDLER_ENTRY_SIZE * Iop::CIntc::LINES::LINE_CDROM)]);

```

Comment:
Please move this to a function. Something like `PopulateSystemIntcHandlers`.

Also, please use the `SYSTEM_INTRHANDLER` structure to set the `handler` member to 1. 

Would also be good to add the link to the poweroff.c line on PS2SDK's GitHub repo that shows why we need that.